src/test/groovy/com/anotherchrisberry/spock/extensions/retry/RetryOnFailureOnlyExclude.groovy

```diff
@@ -0,0 +1,32 @@
+package com.anotherchrisberry.spock.extensions.retry
```

Comment:
I'd love to see one more test against `only`, where some other exception is thrown, just to verify that `only` will restrict the retry to the specified classes of exceptions.

Reply:
Done although the remainder of testsuite should be sufficient (no existing test has been broken).